pymemcache/client/base.py

```diff
@@ -867,6 +867,27 @@ def version(self):
             raise MemcacheUnknownError("Received unexpected response: %s" % results[0])
         return after
 
+    def misc(self, command, end_tokens="\r\n"):
```

Comment:
`send_command` definitely sounds better than `arbitrary_command`.
Since it's not a "commonly" used command, I don't think it being slightly more wordy than `get`, `set`, `delete` and so forth.

Reply:
A bit thinking later, `raw_command` might also be suitable. But I don't know that it is considerably better then `send_command`.

Reply:
I like `raw_command`.